### DISCUSSION TITLE
How is inducible clindamycin included in the eucast_rules()

### DISCUSSION THREAD

**Discussion by theanita1** [msberends/AMR#91] [Q&A]
Hi there
For _Staphylococcus_ there is the rule that if "IF resistant to erythromycin AND susceptible to clindamycin, THEN test for inducible MLSB resistance".
My question is, does the package recognise an "inducible clindamycin" variable, and if so, what is the required format? My current data set has both constitutive and inducible variables, the constitutive being an MIC, and the inducible being a POS/NEG character variable. 
Thanks

**Comment by msberends** [comment]
No, this is currently not part of the package. Different types of the same antibiotic (e.g., constitutive and inducible) and different concentrations of the same antibiotic are not supported. 

You this be solved with an `if`/`else` solution in your analysis/report?

**Reply by theanita1** [reply]
Yes, I did a case_when statement
